File: 3379-ScoreOfAString/3379-ScoreOfAString.js
// Last updated: 3/7/2026, 10:01:59 PM
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
    let num = 0;
    let sum = 0;

    for(let i=1; i<s.length; i++){
        let a = s.charCodeAt(i);
        let b = s.charCodeAt(i-1);
        num = a - b;
       
        
        if(num < 0){
            num = -num;
        }
        sum = sum + num;
    }
    return sum;
};

File: 3379-ScoreOfAString/3379-ScoreOfAString.cpp
// Last updated: 8/13/2026, 10:37:49 AM
class Solution {
public:
    int scoreOfString(string s) {
        int sum = 0;

        for(int i=0; i<s.size()-1; i++){
            int diff = abs(s[i]-s[i+1]);
            sum = sum  + diff;
        }
        return sum;
    }
};